Makefile

```diff
@@ -132,6 +132,14 @@ test:
 lint:
 	@golangci-lint --timeout=5m run
 
+# Activate pre-commit hooks
+pre-commit:
+	pre-commit install -t pre-commit -t commit-msg
```

Comment:
If I change a commit message during rebase, will all hooks be executed on only message check?

Reply:
No, it should not.